### PR TITLE
Address all SFI ROPC concerns.

### PR DESCRIPTION
**Pull request by IEvangelist**
## Summary

Address all SFI ROPC concerns.

Contributes to https://github.com/dotnet/docs-aspire/issues/1594

---

#### Internal previews

<details><summary><strong>Toggle expand/collapse</strong></summary><br/>

| 📄 File | 🔗 Preview link |
|:--|:--|
| [docs/architecture/maui/authentication-and-authorization.md](https://github.com/dotnet/docs/blob/63eec49d0378996bb4a38e98c9a7a3ae0dad52ed/docs/architecture/maui/authentication-and-authorization.md) | [docs/architecture/maui/authentication-and-authorization](https://review.learn.microsoft.com/en-us/dotnet/architecture/maui/authentication-and-authorization?branch=pr-en-us-42559) |
| [docs/architecture/maui/mvvm.md](https://github.com/dotnet/docs/blob/63eec49d0378996bb4a38e98c9a7a3ae0dad52ed/docs/architecture/maui/mvvm.md) | [docs/architecture/maui/mvvm](https://review.learn.microsoft.com/en-us/dotnet/architecture/maui/mvvm?branch=pr-en-us-42559) |
| [docs/architecture/microservices/docker-application-development-process/docker-app-development-workflow.md](https://github.com/dotnet/docs/blob/63eec49d0378996bb4a38e98c9a7a3ae0dad52ed/docs/architecture/microservices/docker-application-development-process/docker-app-development-workflow.md) | [docs/architecture/microservices/docker-application-development-process/docker-app-development-workflow](https://review.learn.microsoft.com/en-us/dotnet/architecture/microservices/docker-application-development-process/docker-app-development-workflow?branch=pr-en-us-42559) |
| [docs/architecture/microservices/implement-resilient-applications/implement-resilient-entity-framework-core-sql-connections.md](https://github.com/dotnet/docs/blob/63eec49d0378996bb4a38e98c9a7a3ae0dad52ed/docs/architecture/microservices/implement-resilient-applications/implement-resilient-entity-framework-core-sql-connections.md) | [docs/architecture/microservices/implement-resilient-applications/implement-resilient-entity-framework-core-sql-connections](https://review.learn.microsoft.com/en-us/dotnet/architecture/microservices/implement-resilient-applications/implement-resilient-entity-framework-core-sql-connections?branch=pr-en-us-42559) |
| [docs/architecture/microservices/implement-resilient-applications/monitor-app-health.md](https://github.com/dotnet/docs/blob/63eec49d0378996bb4a38e98c9a7a3ae0dad52ed/docs/architecture/microservices/implement-resilient-applications/monitor-app-health.md) | [docs/architecture/microservices/implement-resilient-applications/monitor-app-health](https://review.learn.microsoft.com/en-us/dotnet/architecture/microservices/implement-resilient-applications/monitor-app-health?branch=pr-en-us-42559) |
| [docs/architecture/microservices/microservice-ddd-cqrs-patterns/cqrs-microservice-reads.md](https://github.com/dotnet/docs/blob/63eec49d0378996bb4a38e98c9a7a3ae0dad52ed/docs/architecture/microservices/microservice-ddd-cqrs-patterns/cqrs-microservice-reads.md) | [docs/architecture/microservices/microservice-ddd-cqrs-patterns/cqrs-microservice-reads](https://review.learn.microsoft.com/en-us/dotnet/architecture/microservices/microservice-ddd-cqrs-patterns/cqrs-microservice-reads?branch=pr-en-us-42559) |
| [docs/architecture/microservices/microservice-ddd-cqrs-patterns/nosql-database-persistence-infrastructure.md](https://github.com/dotnet/docs/blob/63eec49d0378996bb4a38e98c9a7a3ae0dad52ed/docs/architecture/microservices/microservice-ddd-cqrs-patterns/nosql-database-persistence-infrastructure.md) | [docs/architecture/microservices/microservice-ddd-cqrs-patterns/nosql-database-persistence-infrastructure](https://review.learn.microsoft.com/en-us/dotnet/architecture/microservices/microservice-ddd-cqrs-patterns/nosql-database-persistence-infrastructure?branch=pr-en-us-42559) |
| [docs/architecture/microservices/multi-container-microservice-net-applications/data-driven-crud-microservice.md](https://github.com/dotnet/docs/blob/63eec49d0378996bb4a38e98c9a7a3ae0dad52ed/docs/architecture/microservices/multi-container-microservice-net-applications/data-driven-crud-microservice.md) | [docs/architecture/microservices/multi-container-microservice-net-applications/data-driven-crud-microservice](https://review.learn.microsoft.com/en-us/dotnet/architecture/microservices/multi-container-microservice-net-applications/data-driven-crud-microservice?branch=pr-en-us-42559) |
| [docs/architecture/microservices/multi-container-microservice-net-applications/test-aspnet-core-services-web-apps.md](https://github.com/dotnet/docs/blob/63eec49d0378996bb4a38e98c9a7a3ae0dad52ed/docs/architecture/microservices/multi-container-microservice-net-applications/test-aspnet-core-services-web-apps.md) | [docs/architecture/microservices/multi-container-microservice-net-applications/test-aspnet-core-services-web-apps](https://review.learn.microsoft.com/en-us/dotnet/architecture/microservices/multi-container-microservice-net-applications/test-aspnet-core-services-web-apps?branch=pr-en-us-42559) |
| [docs/core/extensions/configuration-providers.md](https://github.com/dotnet/docs/blob/63eec49d0378996bb4a38e98c9a7a3ae0dad52ed/docs/core/extensions/configuration-providers.md) | [docs/core/extensions/configuration-providers](https://review.learn.microsoft.com/en-us/dotnet/core/extensions/configuration-providers?branch=pr-en-us-42559) |
| [docs/orleans/grains/grain-persistence/index.md](https://github.com/dotnet/docs/blob/63eec49d0378996bb4a38e98c9a7a3ae0dad52ed/docs/orleans/grains/grain-persistence/index.md) | [docs/orleans/grains/grain-persistence/index](https://review.learn.microsoft.com/en-us/dotnet/orleans/grains/grain-persistence/index?branch=pr-en-us-42559) |
| [docs/orleans/host/configuration-guide/configuring-ado-dot-net-providers.md](https://github.com/dotnet/docs/blob/63eec49d0378996bb4a38e98c9a7a3ae0dad52ed/docs/orleans/host/configuration-guide/configuring-ado-dot-net-providers.md) | [docs/orleans/host/configuration-guide/configuring-ado-dot-net-providers](https://review.learn.microsoft.com/en-us/dotnet/orleans/host/configuration-guide/configuring-ado-dot-net-providers?branch=pr-en-us-42559) |
| [docs/orleans/implementation/streams-implementation/azure-queue-streams.md](https://github.com/dotnet/docs/blob/63eec49d0378996bb4a38e98c9a7a3ae0dad52ed/docs/orleans/implementation/streams-implementation/azure-queue-streams.md) | [docs/orleans/implementation/streams-implementation/azure-queue-streams](https://review.learn.microsoft.com/en-us/dotnet/orleans/implementation/streams-implementation/azure-queue-streams?branch=pr-en-us-42559) |

</details>

### CODE REVIEW

File: docs/architecture/maui/authentication-and-authorization.md [hunk 1]
```diff
@@ -147,33 +152,37 @@ public static IEnumerable<Client> GetClients(Dictionary<string,string> clien
...
+                "webhooks"
+            },
+            //Allow requesting refresh tokens for long lived API access
+            AllowOfflineAccess = true,
+            AllowAccessTokensViaBrowser = true,
+            AlwaysIncludeUserClaimsInIdToken = true,
             AccessTokenLifetime = 60 * 60 * 2, // 2 hours
-            IdentityTokenLifetime = 60 * 60 * 2 // 2 hours
-        }
+            IdentityTokenLifetime= 60 * 60 * 2 // 2 hours
```
Comment (gewarren) [inline docs/architecture/maui/authentication-and-authorization.md:184]:
```suggestion
            IdentityTokenLifetime = 60 * 60 * 2 // 2 hours
```

File: docs/architecture/microservices/docker-application-development-process/docker-app-development-workflow.md [hunk 2]
```diff
@@ -379,12 +379,14 @@ services:
...
     image: mcr.microsoft.com/mssql/server:latest
     environment:
-      - SA_PASSWORD=Pass@word
+      - SA_PASSWORD=[PLACEHOLDER]
       - ACCEPT_EULA=Y
     ports:
       - "5433:1433"
 ```
 
+[!INCLUDE [managed-identities](../../../framework/includes/managed-identities.md)]
```
Comment (gewarren) [inline docs/architecture/microservices/docker-application-development-process/docker-app-development-workflow.md:388]: Does this make sense here? In a later file, you didn't add the include.

**Review by gewarren — APPROVED** [review]
I should probably move that include file out of the framework folder.

File: docs/architecture/microservices/docker-application-development-process/docker-app-development-workflow.md (hunk 2 above)
Comment (IEvangelist) [reply docs/architecture/microservices/docker-application-development-process/docker-app-development-workflow.md:388]: Where exactly? I did omit it from several of the other's as they seemed to talk about local deployments, not cloud related.

File: docs/architecture/microservices/multi-container-microservice-net-applications/test-aspnet-core-services-web-apps.md [hunk 3]
```diff
@@ -176,7 +176,7 @@ services:
       - "5672:5672"
   sqldata:
     environment:
-      - SA_PASSWORD=Pass@word
+      - SA_PASSWORD=[PLACEHOLDER]
```
Comment (gewarren) [inline docs/architecture/microservices/multi-container-microservice-net-applications/test-aspnet-core-services-web-apps.md:179]: @IEvangelist Here.

**Comment by IEvangelist** [comment]
> I should probably move that include file out of the framework folder.

Yeah, I was thinking that too.

**Pull request merged**